runcellpose.py

```diff
@@ -31,6 +31,7 @@
 may take some time. If you want to use a GPU to run the model, you'll need a compatible version of PyTorch and a 
 supported GPU. Instructions are avaiable at this link: {CUDA_LINK}
 
+To use Omnipose models, and mask reconstruction method you'll want to run 'pip install omnipose'.

```

Comment:
Can you add a bit more detail here? We don't need a ton, but something like this (stuff in {brackets} is to be filled in  "This module now also supports Ominpose. Omnipose is different from Cellpose in 2 ways: the models themselves {a few words about how these are different from regular Cellpose} and the masking {what is this and why do we care?}.  You can mix and match Omnipose models with Cellpose style masking or vice versa {I don't know if this is true, is it?}, {if we have any recommendations based on our experience and/or what the paper says, this is a good place for them, otherwise say something like "you should try a few combinations to see what works best for your particular data}"